src/contexts/dataContext.js

```diff
@@ -1,30 +1,119 @@
 import { createContext, useContext, useEffect, useReducer } from "react";
+
 import { dataReducerFunc, initialState } from "../reducer/dataReducer";
-import axios from "axios";
+import {
+  getData,
+  getCartData,
+  getWishlistData,
+  postCartData,
+  postWishlistData,
+} from "../utils/utilDataFuncs";
+
+import {
+  removeFromCartUtilFunc,
+  removeFromWishlistUtilFunc,
```

Comment:
large names are good but remember we should not exceed a certain value (26-32) and can pick any threshold as per your choice. 
No problem is there in your code. I mentioned it just for discussing

Reply:
Thanks will keep them in mind next time :)